content/docs/introducing-jsx.md

```diff
@@ -68,88 +68,88 @@ ReactDOM.render(
 );
 ```
 
-[](codepen://introducing-jsx)
+[Przetestuj kod na CodePen](codepen://introducing-jsx)
 
-We split JSX over multiple lines for readability. While it isn't required, when doing this, we also recommend wrapping it in parentheses to avoid the pitfalls of [automatic semicolon insertion](https://stackoverflow.com/q/2846283).
+Dla czytelności podzieliliśmy kod JSX na kilka linii. Nie jest to wprawdzie wymagane, ale polecamy również otaczanie go w nawiasy. Pozwoli to uniknąć pułapek związanych z [automatycznym wstawianiem średników (ang.)](http://stackoverflow.com/q/2846283) w JavaScripcie.
 
-### JSX is an Expression Too {#jsx-is-an-expression-too}
+### JSX również jest wyrażeniem {#jsx-is-an-expression-too}
 
-After compilation, JSX expressions become regular JavaScript function calls and evaluate to JavaScript objects.
+Po kompilacji, wyrażenia JSX-owe stają się zwykłymi wywołaniami funkcji w JavaScripcie i są ewaluowane do obiektów javascriptowych.
 
-This means that you can use JSX inside of `if` statements and `for` loops, assign it to variables, accept it as arguments, and return it from functions:
+Oznacza to, że możesz używać znaczników JSX również wewnątrz instrukcji warunkowych `if`, pętli `for`, przypisywać je do zmiennych, oczekiwać przekazania jako argumenty do funkcji, zwracać z funkcji i w wielu innych miejscach.
 
 ```js{3,5}
 function getGreeting(user) {
   if (user) {
-    return <h1>Hello, {formatName(user)}!</h1>;
+    return <h1>Witaj, {formatName(user)}!</h1>;
   }
-  return <h1>Hello, Stranger.</h1>;
+  return <h1>Witaj, nieznajomy.</h1>;
 }
 ```
 
-### Specifying Attributes with JSX {#specifying-attributes-with-jsx}
+### Podawanie atrybutów w JSX {#specifying-attributes-with-jsx}
 
-You may use quotes to specify string literals as attributes:
+Możesz przekazać literał znakowy jako atrybut, używając poniższej składni:
```

Comment:
```suggestion
Możesz przekazać literał ciągu znaków jako atrybut, używając poniższej składni:
```

Reply:
W tym wypadku trzymał bym sie przyjętego nazewnictwa i pozostał przy literał znakowy - jest to nazwa zrozumiała i używana przez wszystkich więc nie ma co jej zbytnio komplikowac.

https://pl.wikipedia.org/wiki/Litera%C5%82_znakowy